vantage/manifest.json

```diff
@@ -0,0 +1,41 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "f9d211f9-8cd2-4bab-9718-462c5a3fa9ad",
+  "app_id": "vantage",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "This integration allows Datadog users to import their Datadog costs into Vantage and track alongside their other infrastructure spend.",
+    "title": "Vantage",
+    "media": [
+      {
+        "media_type": "image",
```

Comment:
For the images, would you be able to place them on a background or take new screenshots so that they have a 16:9 aspect ratio? This will ensure that they don't get cut off in the media carousel. More guidelines on media can be found [here](https://docs.datadoghq.com/developers/marketplace/#media-carousel-requirements).